spec/amp-3p-video.md

```diff
@@ -5,7 +5,7 @@ The generic [`amp-video`](https://go.amp.dev/c/amp-video) component plays videos
 Unfortunately, not all videos can be embedded this way. For these specialized cases, AMP provides vendor-optimized components like [`amp-youtube`](https://go.amp.dev/c/amp-youtube), [`amp-ima-video`](https://go.amp.dev/c/amp-ima-video), and [others](./amp-video-interface.md).
 Internally these players load an iframe whose page communicates with the outer document to coordinate playback.
 
-We would prefer to not have additional custom video implementations. Instead we believe, [`amp-video-iframe`](https://go.amp.dev/c/amp-video-iframe) should be used instead, since it is a generic component that can load any video document to coordinate playback.
+**We would prefer to not have additional custom video implementations.** Instead we believe, [`amp-video-iframe`](https://go.amp.dev/c/amp-video-iframe) should be used instead, since it is a generic component that can load any video document to coordinate playback.
```

Comment:
Same as previous comment about ordering documentation. Can we also change the wording to something more opinionated? Like:
`We are no longer accepting custom video implementations in the form of AMP components. Instead, new implementations should com int he form of integrations with amp-video-iframe. If you believe you are unable to integrate with amp-video-iframe something something contact here?`

Reply:
Added blurb about contact.

How do you see this being better restructured? Unsure about what the sub-sections are here.

About the wording with `we are no longer accepting custom video implementations in the form of AMP components`, that's stronger than the current policy (we're considering #26572 as an alternative, for example)